Observing-Interface-Setup/efd-kafka.rst

```diff
@@ -0,0 +1,89 @@
+#############
+EFD SAL Kafka 
+#############
+
+Overview
+^^^^^^^^
+This topic covers information on how to perform actions on the EFD SAL Kafka producers/writers on the non-main domains.
```

Comment:
what is a non-main domain?

Reply:
The adopted nomenclature is production/non-production DDS domains. May want to point to https://tstn-023.lsst.io/#dds-domain-and-dds-partitions